src/decoder/lattice-faster-decoder.h

```diff
@@ -368,6 +407,9 @@ class LatticeFasterDecoderTpl {
   //    Elem *tail;
   //  };
 
+  // Deletes the elements of the singly linked list tok->links.
+  inline static void DeleteForwardLinks(Token *tok);
```

Comment:
I do not really understand the use of `inline` on a forward declaration of a class member function. `inline` allows the function to have multiple identical definitions, but it's not defined inline anywhere. It seems redundant, unless I am really missing something?

Reply:
It's not a new method whose definition in its cpp file. Or you mean that we should remove ```inline``` in the declaration.